README.md

```diff
@@ -64,7 +64,7 @@ almost never be shared between different users.
 
 ## Versioning
 
-Each version of the client is tagged and the version is updated accordingly.
+Each version of the client is tagged, and the version is updated accordingly.
```

Comment:
I don't believe there is a comma before and unless it's a list (https://www.grammarly.com/blog/comma-before-and/?&utm_source=google&utm_medium=cpc&utm_campaign=11862361103&utm_targetid=dsa-1233402314764&gclid=Cj0KCQjwlMaGBhD3ARIsAPvWd6jgvmSCZvouUFNLBu0e83D_I8ejZkged-WhvJtcnxcHUKRGGwr2l0QaAhmbEALw_wcB&gclsrc=aw.ds)